lib/movable_ink/aws/metadata.rb

```diff
@@ -26,6 +27,20 @@ def retrieve_metadata(key, tries: 3)
         raise MovableInk::AWS::Errors::MetadataTimeout
       end
 
+      def retrieve_metadata(key)
+        retrieve_data("/latest/meta-data/#{key}")
+      end
```

Comment:
Can you fix this to allow `tries`? To ensure the method remains backwards compatible

Reply:
Sure! Good catch, I'll fix it.

Reply:
@aackerman please review once you have a chance, I've just made changes according to your comment.